### PR TITLE
Updates for master.

### PR DESCRIPTION
**Pull request by razyke**
Removed not using imports, added more javadoc descriptions, fixed login logic, also remake tests.

### CODE REVIEW

File: src/main/java/services/AuthService.java [hunk 1]
```diff
@@ -71,7 +69,7 @@
         }
 
         if (!error) {
-            //Check on unique values in DB
+            //Check on unique values in DB.
             if (dao.isExist("login", user.getLoginName())) {
```
Comment (ginsenghillock) [inline src/main/java/services/AuthService.java]: Do you use only one DAO for all entities? It'll be better to split it up

Comment (razyke) [reply src/main/java/services/AuthService.java]: Ok, we will split.
About checking unique values, is this ok that we send in method with 2 strings, one is colum in table, another value. Or better create each method for each columns?

Comment (ginsenghillock) [reply src/main/java/services/AuthService.java]: it'll be better to create separate methods

**Comment by razyke** [comment]
Seperated dao and methods, has been taken from branch "workWithDB".

**Comment by ginsenghillock** [comment]
Reviewed

**Pull request merged**